tests/feature/console/suseconnect.pm

```diff
@@ -0,0 +1,36 @@
+# Feature tests
+#
+# Copyright © 2016 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Case #1480023 : [316585] Drop suseRegister
+
+use base "consoletest";
+use testapi;
+
+sub run() {
+    my $self = shift;
+
+    select_console 'root-console';
+
+    #Check SUSEConnect is installed
+    script_run("rpm -q SUSEConnect | tee /dev/$serialdev");
+    wait_serial("SUSEConnect") || die "rpm -q SUSEConnect failed";
```

Comment:
what I would do to verify it's there is plainly assert_script_run('rpm -q SUSEConnect');